package/use-ruby-2.5-in-rmt-data-import.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/bin/rmt-data-import b/bin/rmt-data-import
-index 600a2a4..2caca12 100755
+index f232051..8b4330a 100755
 --- a/bin/rmt-data-import
 +++ b/bin/rmt-data-import
 @@ -1,4 +1,4 @@
```

Comment:
Just FYI, you can also hack it so that it doesn't care about the rest of the file, i.e.:
```
 @@ -1,2 +1,2 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby.ruby2.5

```

Reply:
yes but this was just faster :)